tests-build/tests/fail/macros_invalid_input.stderr

```diff
@@ -93,3 +93,21 @@ note: the lint level is defined here
    |
 1  | #![deny(duplicate_macro_attributes)]
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: second test attribute is supplied, try to order it after `tokio::test`
+  --> $DIR/macros_invalid_input.rs:49:1
+   |
+49 | #[::core::prelude::v1::test::test]
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
What? The `#[tokio::test]` macro is first, so what do you mean by "after"?

Reply:
I supposed this to be generated by other preceding test macros, e.g. `#[test_log::test]`. I guess I should reshape it or keep it origin. Any preference ?

Reply:
How about "second test attribute is supplied, consider removing or ordering it after `tokio::test`" ?

Reply:
My challenge with the wording here is that the code that triggers this error has `#[tokio::test]` first, and then afterwards, it has a `#[::core::prelude::v1::test]`. So suggesting that you move `#[::core::prelude::v1::test]` after `#[tokio::test]` doesn't make since it's already after `#[tokio::test]`. Perhaps you could say "above" instead of "after"?

Reply:
I know, it is strange. It should be something similar to following.

```rust
❯ grep -n -B 2 -A 2 "with_append_test_attribute_and_test_args_and_panic_async" tests/mod.rs
84-#[test_log::test]
85-#[tokio::test]
86:async fn with_append_test_attribute_and_test_args_and_panic_async(x: i8, _y: i8) {
87-  assert_eq!(x, 0);
88-}
❯
❯ cargo test with_append_test_attribute_and_test_args_and_panic_async -- --nocapture
error: second test attribute is supplied, consider removing it or ordering it after `tokio::test`
  --> tests/mod.rs:84:1
   |
84 | #[test_log::test]
   | ^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the attribute macro `test_log::test` (in Nightly builds, run with -Z macro-backtrace for more info)

error: could not compile `test-log` (test "mod") due to 2 previous errors

```

Moving it "before" or "above" will result in duplicated test runs.

I guess we could:
1. Introduce dev dependency `test-log`. This ways the words should be more intuitively. Is it worth ?
2. Use the short version "second test attribute is supplied".
3. Other suggestions.

Hi @Darksonn, thank you for reviewing. I have updated this with a fixup commit targeting to your two concerns. Regarding the error words, I am open to any sentence, simply not good at 😮‍💨 .